test/test_model.py

```diff
@@ -36,201 +37,166 @@ def restore_env(env):
         self.addCleanup(restore_env, os.environ.copy())
 
         os.environ['JUJU_UNIT_NAME'] = 'myapp/0'
+        # language=YAML
```

Comment:
does your editor still pick this up if it's _inside_ the yaml?

Reply:
it doesn't, unfortunately, but I removed most of these, so I'll remove this one as well.